### ISSUE TITLE
FreePBX user syncing fails on 7.4 with AD user provider

### ISSUE THREAD

**Issue by Stell0** [NethServer/dev#5361]
**Steps to reproduce**

- Install NethServer 7.4 with PBX
- Choose AD as user provider
  [event will fail]
- manually launch `/usr/bin/scl enable rh-php56 -- /usr/sbin/fwconsole userman --syncall --force --verbose`

**Expected behavior**

- FreePBX users synced with NethServer users 

**Actual behavior**

- verify that it returns "Invalid credentials"

**Components**

- NethServer 7.4
- NethServer PBX
- nethserver-freepbx-14.0.0-1.ns7.noarch

**Other Infos**

NethServer 7.4 machine account has unicode password, ` nethserver-freepbx-conf-users` writes it in wrong encoding. Writing unicode password from FreePBX interface, it's encoded like "\u3bce\u2e0b\uaee4\u7dcd\u7949\ue756\ubbad"
you can see saved password with `mysql> select val from kvstore_FreePBX_modules_Userman`

**Comment by Stell0** [comment]
**Test case 1**
- Install NethServer 7.4 with PBX
- Choose AD as user provider
- manually launch '/usr/bin/scl enable rh-php56 -- /usr/sbin/fwconsole userman --syncall --force --verbose'
- Verify that it doesn't fail

**Test case 2**
- Install NethServer 7.4 with PBX
- Choose LDAP as user provider
- manually launch '/usr/bin/scl enable rh-php56 -- /usr/sbin/fwconsole userman --syncall --force --verbose'
- Verify that it doesn't fail

**Comment by Stell0** [comment]
nethserver-testing: nethserver-freepbx-14.0.0-1.7.g9ebc5c9.ns7.noarch.rpm

**Comment by gsanchietti** [comment]
After switching from AD to LDAP, the event fails with the following error:
```
Sep 15 11:01:55 test esmith::event[1586]: Starting Sync on directory 'NethServer AD'...
Sep 15 11:01:55 test esmith::event[1586]: 
Sep 15 11:01:55 test esmith::event[1586]:                               
Sep 15 11:01:55 test esmith::event[1586]:  [Adldap\Auth\BindException]
Sep 15 11:01:55 test esmith::event[1586]:  Invalid DN syntax
Sep 15 11:01:55 test esmith::event[1586]:                               
Sep 15 11:01:55 test esmith::event[1586]: 
Sep 15 11:01:55 test esmith::event[1586]: userman [--syncall] [--sync SYNC] [--force] [--list]
Sep 15 11:01:55 test esmith::event[1586]: 
Sep 15 11:01:55 test esmith::event[1586]: Action: /etc/e-smith/events/nethserver-freepbx-update/S60nethserver-freepbx-conf-users FAILED: 34 [1.497107]
```

Also, on the first run, if the user select a language other than English, FreePBX blocks with "Proxy error" and there is no way to recover it.

**Comment by gsanchietti** [comment]
In nethserver-testing:
nethserver-freepbx-14.0.0-1.9.gbe7ba69.ns7.noarch.rpm

**Comment by gsanchietti** [comment]
Released in 7.4/nethserver-base:
nethserver-freepbx-14.0.1-1.ns7.noarch.rpm
